.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        rust: [stable, nightly]
```

Comment:
I'm not sure about running nightly on all platforms, ubuntu should be enough, also we should allow it to fail